src/components/views/rooms/MemberInfo.js

```diff
@@ -73,6 +73,11 @@ module.exports = React.createClass({
     },
 
     componentDidMount: function() {
+        // only display the devices list if our client supports E2E *and* the
+        // feature is enabled in the user settings
+        this._enableDevices = MatrixClientPeg.get().isCryptoEnabled() &&
+            UserSettingsStore.isFeatureEnabled("e2e_encryption");
+
```

Comment:
This should probably be in componentWillMount really, as I think you'll be be testing it and getting `undefined` on the first render call.